doc/cjdns.md

```diff
@@ -23,17 +24,37 @@ somewhat centralized. I2P connections have a source address and I2P is slow.
 CJDNS is fast but does not hide the sender and the recipient from intermediate
 routers.
 
-## Installing CJDNS and connecting to the network
+## Installing CJDNS and finding a friend to connect to the network
```

Comment:
Should it still be called a friend if it's simply a random active public peer selected from a list? :smile:

Reply:
Kind of like a dating app? :smile: